examples/kubernetes/manifests/registry.yml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: registry
+  labels:
+    app: registry
+    tier: registry
+spec:
+  type: NodePort
+  # type: LoadBalancer
+  ports:
+  - port: 5000
+    targetPort: 5000
+  selector:
+    app: registry
+    tier: registry
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: registry
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: registry
+        tier: registry
+    spec:
+      containers:
+      - name: registry
+        image: library/registry:2.6
+        command: ["/bin/sh", "/etc/docker/registry/init"]
```

Comment:
Not quite. This init script will also install self-signed certificates. Something that we wouldn't have on a serious deployment, but that it's needed in simple deployments that are testing things out. Maybe I should add a comment on that as well.